crates/apub/src/activities/following/follow.rs

```diff
@@ -61,7 +62,8 @@ impl Follow {
       .ok();
 
     let follow = Follow::new(actor, community, context)?;
-    let inbox = vec![community.shared_inbox_or_inbox()];
+    // todo: this should probably filter and only send if the community is remote?
+    let inbox = ActivitySendTargets::to_inbox(community.shared_inbox_or_inbox());
```

Comment:
Also not necessary to check here.

Reply:
see https://github.com/LemmyNet/lemmy/pull/3605/files/6e445ac581c9336a90698e9e2ae8e44ca32f57e9#diff-e93ff81a1ea97b27fc17a51bfb546adb7b43c49c39269ea7b2e90c8e62c80bb1